app/src/main/java/org/oppia/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -273,6 +257,14 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
     )
   }
 
+  fun markHomeMenuCloseDrawer() {
+    binding.fragmentDrawerNavView.menu.getItem(
+      NavigationDrawerItem.HOME.ordinal
```

Comment:
home is not always an option, the user might click on switch profile from any menu screen like option or help, creating a different issue for this.

Reply:
@anandwana001 Please reference issue number here.

Reply:
Also, if that is the case then shouldn't `markHomeMenuCloseDrawer ` be renamed to something else which does not include `Home`

Reply:
#1762 is the issue number and the name can be change in that issue as this complety solves there, changing name here might create confusion.